tests/data/xref/inline-idl-enum.json

```diff
@@ -42,12 +42,12 @@
       "term": "imports",
       "types": ["enum-value"],
       "for": "ServiceWorkerUpdateViaCache",
-      "id": "9a7f11d8f9a46fce8d14170914f73271d1cc8896"
+      "id": "322b86ae2a57f3f9c9808ed6eef392ad921a66db"
```

Comment:
Updating these ids will be a pain in future. Can we mock the id generating function in tests or generate stable ids during development mode?

Reply:
Yes please... this drove me insane.

Reply:
We can use a `static` counter as ID if in _test_ (will `window.__karma__` work?).

Reply:
I've never used that, so I'm not sure... but happy to try in a follow up.